manager/state/raft/raft.go

```diff
@@ -914,7 +936,7 @@ func (n *Node) configure(ctx context.Context, cc raftpb.ConfChange) error {
 	case <-ctx.Done():
 		n.wait.trigger(cc.ID, nil)
 		return ctx.Err()
-	case <-n.stopCh:
+	case <-n.Ctx.Done():
```

Comment:
So what is the preferred approach here? Should I revert to the first way of doing it? (using the flag?)